acceptance/lib/helper.rb

```diff
@@ -193,6 +193,7 @@ def puppet_apply_as_puppet_user
     # installed.
     manifest_path = master.tmpfile("puppetserver_manifest.pp")
     herp_path = master.tmpfile("herp")
+    bin_dir = "/opt/puppetlabs/agent/bin/"
```

Comment:
D'oh! I the same value is also in beaker 2.2.0:
https://github.com/puppetlabs/beaker/blob/beaker2.2.0/lib/beaker/host/unix.rb#L61

I just know that 2.3.0 includes additonal AIO magic.